net-libs/libp8_platform/libp8_platform-2.1.0.1.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="Platform support library used by libCEC and binary add-ons for Kodi"
+DESCRIPTION="This library provides platform specific support for other libraries"
+HOMEPAGE="http://libcec.pulse-eight.com/"
+COPYRIGHT="2011-2015 Pulse-Eight Limited"
+LICENSE="GNU GPL v2"
+REVISION="1"
+
+SOURCE_URI="https://github.com/Pulse-Eight/platform/archive/p8-platform-$portVersion.tar.gz"
+CHECKSUM_SHA256="064f8d2c358895c7e0bea9ae956f8d46f3f057772cb97f2743a11d478a0f68a0"
+SOURCE_DIR="platform-p8-platform-$portVersion"
+
+ARCHITECTURES="x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libp8_platform$secondaryArchSuffix = $portVersion
+	lib:libp8_platform$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+PROVIDES_devel="
+	libp8_platform${secondaryArchSuffix}_devel = $portVersion
+	devel:libp8_platform$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	libp8_platform$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+	cmake -DCMAKE_INSTALL_PREFIX=$prefix ..
```

Comment:
Checked this on gcc2hybrid?

Reply:
There is no x86_gcc2 in ARCHITECTURE.

Reply:
You have SECONDARY_ARCHITECTURES defined, for this libraries end up in $libDir instead of $prefix/lib

Reply:
-DCMAKE_INSTALL_LIBDIR=$libDir also seems to have no effect.

Reply:
```diff
-	cmake -DCMAKE_INSTALL_PREFIX=$prefix ..
+	cmake -DCMAKE_BUILD_TYPE=Release  \
+		cmake -DCMAKE_INSTALL_PREFIX=$prefix \
+		cmake -DCMAKE_INSTALL_LIBDIR=$libDir ..
```